docs/badges/details.rst

```diff
@@ -0,0 +1,168 @@
+Implementation Details
+======================
+
+Badges feature is implemented as a separate application within the Credentials Open edX service:
+
+.. code-block::
+
+        credentials/apps/badges/
+
+Events (Event Bus)
+------------------
+
+        Badges feature always relies on events (public signals) with user data.
+
+New events
+~~~~~~~~~~
+
+Badges feature extends the set of already `published events`_ with its own set of new public events:
+
+- `BADGE_AWARDED` (Credentials)
+- `BADGE_REVOKED` (Credentials)
```

Comment:
Have the new events been proposed or accepted yet?

Reply:
These two are planned as a part of the https://github.com/openedx/openedx-events/pull/303. 
WIP.